node/blockchain/inner_models/signed_change_request/coin_transfer.py

```diff
@@ -7,8 +9,20 @@ class CoinTransferSignedChangeRequest(SignedChangeRequest):
 
     def validate_business_logic(self):
         super().validate_business_logic()
-        # TODO(dmu) CRITICAL: Implement in https://thenewboston.atlassian.net/browse/BC-217
+        self.validate_circular_transactions()
 
     def validate_blockchain_state_dependent(self, blockchain_facade):
         super().validate_blockchain_state_dependent(blockchain_facade)
-        # TODO(dmu) CRITICAL: Implement in https://thenewboston.atlassian.net/browse/BC-217
+        self.validate_amount()
```

Comment:
`self.validate_amount(blockchain_facade)`

Reply:
You, are right. Let me fix it.